pkg/antctl/definitions.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package antctl
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/antctl/handlers"
+	"github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned/scheme"
+	"github.com/vmware-tanzu/antrea/pkg/version"
+)
+
+type VersionResponse struct {
+	handlers.ComponentVersionResponse `json:",inline" yaml:",inline"`
+	AntctlVersion                     string `json:"antctlVersion,omitempty" yaml:"antctlVersion,omitempty"`
+}
+
+func versionTransform(reader io.Reader, single bool) (interface{}, error) {
+	b, err := ioutil.ReadAll(reader)
+	klog.Infof("version transform received: %s", string(b))
+	if err != nil {
+		return nil, err
+	}
+	var cv handlers.ComponentVersionResponse
+	err = json.Unmarshal(b, &cv)
+	if err != nil {
+		return nil, err
+	}
+	return &VersionResponse{
+		ComponentVersionResponse: cv,
+		AntctlVersion:            version.GetFullVersion(),
+	}, nil
+}
+
+// Definition defines command related options of the antctl.
+var Definition = &CommandBundle{
```

Comment:
should this be just `CommandList` instead of `Definition`?

Reply:
Or how about just call it `Bundle`?

Reply:
Changed to `commandList`.